src/test/java/teammates/test/cases/ui/InstructorCourseAddActionTest.java

```diff
@@ -115,34 +108,63 @@ public void testExecute() throws Exception{
+        addAction = getAction(Const.ParamsNames.USER_ID, instructorId,
+                              Const.ParamsNames.COURSE_ID, "ticac.tpa2.id",
+                              Const.ParamsNames.COURSE_NAME, "ticac tpa2 name");
         pageResult = (ShowPageResult) addAction.executeAndPostProcess();
         
         String expectedDestination = Const.ViewURIs.INSTRUCTOR_COURSES + "?error=false&user=idOfInstructor1OfCourse1";
         assertEquals(expectedDestination, pageResult.getDestinationWithParams());
         assertEquals(false, pageResult.isError);
-        String expectedStatus = "The course has been added. Click <a href=\"/page/instructorCourseEnrollPage?courseid=ticac.tpa2.id&user=idOfInstructor1OfCourse1\">here</a> to add students to the course or " +
-                        "click <a href=\"/page/instructorCourseEditPage?courseid=ticac.tpa2.id&user=idOfInstructor1OfCourse1\">here</a> to add other instructors.<br>If you don't see the course in the list below, please refresh the page after a few moments.";
+        String expectedStatus = "The course has been added. Click <a href=\"/page/instructorCourseEnrollPage?"
+                                + "courseid=ticac.tpa2.id&user=idOfInstructor1OfCourse1\">here</a> to add students "
+                                + "to the course or click <a href=\"/page/instructorCourseEditPage?"
+                                + "courseid=ticac.tpa2.id&user=idOfInstructor1OfCourse1\">here</a> to add other "
+                                + "instructors.<br>If you don't see the course in the list below, please refresh "
+                                + "the page after a few moments.";
         assertEquals(expectedStatus, pageResult.getStatusMessage());
         
-        pageData = (InstructorCoursesPageData)pageResult.data;
+        pageData = (InstructorCoursesPageData) pageResult.data;
         assertEquals(1, pageData.allCourses.size());
         
-        expectedLogMessage = "TEAMMATESLOG|||instructorCourseAdd|||instructorCourseAdd" 
-                + "|||true|||Instructor(M)|||Instructor 1 of Course 1" 
-                + "|||idOfInstructor1OfCourse1|||instr1@course1.tmt|||Course added : ticac.tpa2.id<br>Total courses: 1" 
-                + "|||/page/instructorCourseAdd";
+        expectedLogMessage = "TEAMMATESLOG|||instructorCourseAdd|||instructorCourseAdd|||true|||Instructor(M)|||"
+                             + "Instructor 1 of Course 1|||idOfInstructor1OfCourse1|||instr1@course1.tmt|||"
+                             + "Course added : ticac.tpa2.id<br>Total courses: 1|||/page/instructorCourseAdd";
         assertEquals(expectedLogMessage, addAction.getLogMessage());
         
         // delete the new course
         CoursesLogic.inst().deleteCourseCascade("ticac.tpa2.id");
+        
+        ______TS("Test archived Courses");
+        InstructorAttributes instructorOfArchivedCourse = dataBundle.instructors.get("instructorOfArchivedCourse");
```

Comment:
There is a comment at at the start of the file that says that the path for checking archived courses will not be covered, you can remove/modify that comment now that you have added this case